options.go

```diff
@@ -1223,6 +1242,13 @@ const (
 	// ValueStorageLowReadLatency indicates Pebble should prefer storing values
 	// in-place.
 	ValueStorageLowReadLatency
+
+	// ValueStorageLatencyTolerant indicates value retrieval can tolerate
+	// additional latency, so Pebble should aggressively prefer storing values
+	// separately if it can reduce write amplification. Pebble may choose to
+	// separate values under this policy even if they would not otherwise be
+	// separated according to the global Options' ValueSeparationPolicy.
+	ValueStorageLatencyTolerant
```

Comment:
Comment suggests it might separate values even when value separation is disabled globally (I don't think this is the case)